src/main/java/com/stripe/model/terminal/Location.java

```diff
@@ -100,4 +100,29 @@ public static LocationCollection list(Map<String, Object> params, RequestOptions
     String url = String.format("%s%s", Stripe.getApiBase(), "/v1/terminal/locations");
     return requestCollection(url, params, LocationCollection.class, options);
   }
+
```

Comment:
Terminal endpoints below actually would be correct, but it's not consistent. The client metadata now generate static methods

Reply:
Do you think we want to release this as 8.0 too?

Reply:
Yeah, we need to quickly talk about static methods (globally, not just java) because I think this is becoming critical. I also need to release those in the libs so it's on my todo list